site/docs/api.md

```diff
@@ -5,4 +5,4 @@ permalink: /docs/api
 title: API Documenation
 ---
 
-This guide provides web services available to integrate ManageIQ with external applications. It details the specification of the ManageIQ REST API, which is implemented as standard REST HTTP requests and responses of content type JSON.
+In this guide you will find a detailed overview of REST API, reference material, appendices, and plenty of examples. The guides provide information on how to integrate ManageIQ with external applications. It details the specification of the ManageIQ REST API, which is implemented as standard REST HTTP requests and responses of content type JSON.
```

Comment:
Should similar change be made to site/docs/index.md ?

Reply:
Ah. I thought that the text being the same meant it was sourcing from the same file in my feedback above, which apparently is not the case. @JPrause Looks like you'll indeed want to change the separate docs/index.md too.

Your previous edit is still valid though as when you resize the window down to a mobile view the links still won't be on the left 😄